### ISSUE TITLE
GB Mobile Number

### ISSUE THREAD

**Issue by echohive** [formvalidation/support#647]
Hi,

This is probably an easy fix, but my regex skills usually result in me ordering a pizza or summoning the police 😞

A GB mobile number always starts with 07, and is always 11 digits long. At the moment, a 10 digit number will be marked as valid - this is likely because a landline number can be 10 or 11 digits. I'm just wondering if this can be fixed on future versions at all?

Source: [Wikipedia](https://en.wikipedia.org/wiki/Telephone_numbers_in_the_United_Kingdom#Mobile_telephones)

Thanks, and keep up the amazing work!

**Comment by nghuuphuoc** [comment]
You're right. We support 10 digits number because it can cover a landline number.
In this case, you can use the [regexp validator](http://formvalidation.io/validators/regexp/):

```javascript
$(document).ready(function() {
    $('#yourFormId').formValidation({
        framework: 'bootstrap',
        icon: {
            valid: 'glyphicon glyphicon-ok',
            invalid: 'glyphicon glyphicon-remove',
            validating: 'glyphicon glyphicon-refresh'
        },
        fields: {
            mobileNumber: {
                validators: {
                    regexp: {
                        regexp: /^07[3456789]\d{2}\s\d{6}$/i,
                        message: 'The mobile number is not valid'
                    }
                }
            }
        }
    });
});
```

The result:
07145 123456 // invalid (because the right one does not cover 071)
07245 123456 // invalid (because the right one does not cover 072)
07345 123456 // valid
07345123456 // invalid (no space)
07945 123456 // valid
